src/patternfly/components/Dropdown/examples/Dropdown.md

```diff
@@ -63,6 +63,14 @@ import './Dropdown.css'
 {{/dropdown}}
 ```
 
+```hbs title=Menu-item-icons
+{{#> dropdown id="dropdown-align-top-expanded" dropdown--IsActionMenu="true" dropdown--IsExpanded="true" dropdown--HasItemIcons="true" dropdown--HasToggleIcon="true"}}
```

Comment:
needs a different `id` to pass axe testing

Reply:
updated